rollout_ui.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |gem|
   gem.email         = ["jrallison@gmail.com"]
```

Comment:
Should we change email to `developers@trx.tv`?

Reply:
:+1: